src/fetch-location-lambda/fetch-location-handler.test.ts

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios';
+import { handler, emptyLocationInputErrorMessage, invalidResponseObjectErrorMessage } from './fetch-location-handler';
+
+describe('fetch-location-lambda > handler', () => {
+	test('If a simple valid input is passed to the handler, it should return an object with a property "locations", filled with the locations that best fit the given input', async () => {
+		// ----- Arrange -----
+		const mockResponse = {
+			data: {
+				features: [
+					{ place_name: 'Brasília, Federal District, Brazil', center: [-47.8823, -15.7934] },
+					{ place_name: 'Brasília, Cascavel - Paraná, Brazil', center: [-53.42, -24.94] },
+					{ place_name: 'Brasília, Feira De Santana - Bahia, Brazil', center: [-38.9506, -12.2666] },
+					{ place_name: 'Brasília, Arapiraca - Alagoas, Brazil', center: [-36.65, -9.75] },
+					{
+						place_name: 'Brasília Shopping, SCN Qd. 5 Bl. A, Brasília, Federal District 70307, Brazil',
+						center: [-47.889108, -15.786831],
+					},
+				],
+			},
+		};
+
+		// when someone calls axios.get, return mockResponse
+		jest.spyOn(axios, 'get').mockResolvedValue(mockResponse);
```

Comment:
Out of curiosity, what value do you see on these comments for engineers since they already know how to read code?

Reply:
To be completely honest with you, I can't deny that some comments in my may not be so useful... 😅 

In the company that I currently work at, there are a lot of interns, so I ended up getting used to trying to make things as clear as possible, but I totally agree that I should stop abusing the use of comments for the sake of clean code (mostly the more "dummy" ones). 😆